.github/workflows/ci.yml

```diff
@@ -56,8 +56,6 @@ jobs:
     steps:
     - name: CHeckout branch
       uses: actions/checkout@v3
-      with:
```

Comment:
Guess that without this, it breaks on PRs from a fork. Maybe better to just generate the docs on the branch for the release PR.

Reply:
Yeah, but how?